Build/Scripts/runTests.sh

```diff
@@ -243,146 +379,283 @@ if [ ${#INVALID_OPTIONS[@]} -ne 0 ]; then
         echo "-"${I} >&2
     done
     echo >&2
-    echo "${HELP}" >&2
+    echo "call \".Build/Scripts/runTests.sh -h\" to display help and valid options"
     exit 1
 fi
 
-# Move "7.2" to "php72", the latter is the docker container name
-DOCKER_PHP_IMAGE=$(echo "php${PHP_VERSION}" | sed -e 's/\.//')
+handleDbmsOptions
+
+COMPOSER_ROOT_VERSION="5.x.x-dev"
```

Comment:
```suggestion
COMPOSER_ROOT_VERSION="2.x.x-dev"
```

COMPOSER_ROOT_VERSION should be set to the "development" version, which is the next major (when merging this PR **AFTER** current main has been branched away as 1.x.x dev.

Reply:
understood and changed